code/mob/wraith.dm

```diff
@@ -635,6 +798,57 @@
 // Related procs and verbs
 //////////////
 
+/mob/wraith/wraith_decay
+	name = "Plaguebringer"
+	real_name = "plaguebringer"
+	desc = "A terrifyingly bloated spirit."
```

Comment:
Descriptions between subclasses are quite different in tone. Is this intentional?

Reply:
Nope, i havent given enough thought on descriptions, will do so now!